installers/office/installTeams.sh

```diff
@@ -5,3 +5,7 @@ sudo wget -qO /etc/apt/keyrings/teams-for-linux.asc /etc/apt/keyrings/teams-for-
 echo "deb [signed-by=/etc/apt/keyrings/teams-for-linux.asc arch=$(dpkg --print-architecture)] https://repo.teamsforlinux.de/debian/ stable main" | sudo tee /etc/apt/sources.list.d/teams-for-linux-packages.list
 sudo apt update
 sudo apt install teams-for-linux
+
+
+# looking for legacy teams? it's still available!
+# https://web.archive.org/web/20221130115842/https://packages.microsoft.com/repos/ms-teams/pool/main/t/teams/teams_1.5.00.23861_amd64.deb
```

Comment:
@phochstr maybe a good alternative for you? this version works best on my linux mint machine, regarding device frequent audio-device switching and the like...